index.js

```diff
@@ -70,7 +70,14 @@ Filter.prototype.build = function build() {
   var paths = walkSync(srcDir);
 
   this._cache.deleteExcept(paths).forEach(function(key) {
-    fs.unlinkSync(this.cachePath + '/' + key);
+    try {
+      fs.unlinkSync(this.cachePath + '/' + key);
+    } catch (e) {
+      if(e.code !== 'ENOENT')
+      {
```

Comment:
Could you move the curly brace up to the same line as the `if` (to follow with the pattern in the rest of the codebase)?  Sorry to be pedantic here.